test_pytest_timeout.py

```diff
@@ -551,8 +544,9 @@ def custom_trace(*args):
     assert pytest_timeout.is_debugging(custom_trace)
 
 
-def test_not_main_thread(testdir):
-    testdir.makepyfile(
+def test_not_main_thread(pytester):
+    pytest.skip("The 'pytest_timeout.timeout_setup' function no longer exists")
```

Comment:
i'm confused by this, why is there still a test for it then?

Reply:
This was broken in [dd9d6080763c559de1ff5b15deb9c27559dd3431](https://github.com/pytest-dev/pytest-timeout/commit/dd9d6080763c559de1ff5b15deb9c27559dd3431#diff-c1e97e79a9e00c3edb7cbb38156ac2c9567c2ce7101355cda025f38dc97d8009L190) and the test was broad enough that it kept passing.